cachito/web/migrations/versions/84d67d85ed43_add_tidy_compat.py

```diff
@@ -0,0 +1,26 @@
+"""Add tidy_compat option for user requests
```

Comment:
The Request table already has a NxN relation to the Flag table, so I don't think we'll need to add an extra field for that.